alerta/webhooks/grafana.py

```diff
@@ -30,6 +30,8 @@ def parse_grafana(alert, match, args):
     group = args.get('group', 'Performance')
     origin = args.get('origin', 'Grafana')
     service = args.get('service', 'Grafana')
+    timeout = args.get('timeout', 86400)
```

Comment:
This should work, but if you could test it that would be good...
```
timeout = args.get('timeout', current_app.config('ALERT_TIMEOUT'))
```

Reply:
Tested it now. Got and error saying:

2018-08-27 17:38:25,022 - flask.app[93]: ERROR - 'Config' object is not callable [in /venv/lib/python3.6/site-packages/alerta/exceptions.py:95]
Traceback (most recent call last):
  File "/venv/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/venv/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/venv/lib/python3.6/site-packages/flask_cors/decorator.py", line 128, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "/venv/lib/python3.6/site-packages/alerta/auth/decorators.py", line 37, in wrapped
    return f(*args, **kwargs)
  File "/venv/lib/python3.6/site-packages/alerta/webhooks/grafana.py", line 73, in grafana
    incomingAlert = parse_grafana(data, match, request.args)
  File "/venv/lib/python3.6/site-packages/alerta/webhooks/grafana.py", line 33, in parse_grafana
    timeout = args.get('timeout', current_app.config('ALERT_TIMEOUT'))
TypeError: 'Config' object is not callable